{
  "Faults": [
    {
      "latitude": 50.944366648938995,
      "type": "LV",
      "reference": "KU1710",
      "etaUpdate": true,
      "engineerOnSiteTime": "2023-04-15T07:40:00.000+0000",
      "affectedCustomerCount": 5,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "456d9218-6535-4ffc-b94d-d3527be1c5e2",
      "minLongitude": -1.3873,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -1.384535985361074,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-04-15T05:37:00.000+0000",
      "estimatedRestoration": "2023-04-15T22:59:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. We are working hard to get the power back on as quickly as we can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KU1710'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KU1710",
      "maxLongitude": -1.3828,
      "minLatitude": 50.9434,
      "name": "SO16 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "SO16 3DT",
        "SO16 3EH",
        "SO16 3EJ"
      ],
      "location": {
        "coordinates": [
          [
            [
              -1.3873,
              50.9456
            ],
            [
              -1.3828,
              50.9456
            ],
            [
              -1.3828,
              50.9434
            ],
            [
              -1.3873,
              50.9434
            ],
            [
              -1.3873,
              50.9456
            ]
          ]
        ],
        "type": "Polygon"
      },
      "estimatedArrivalOnSiteTime": "2023-04-15T09:30:00.000+0000",
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 50.9456,
      "etrUpdate": true
    },
    {
      "latitude": 51.29518555900868,
      "type": "LV",
      "reference": "KU2009",
      "etaUpdate": true,
      "engineerOnSiteTime": "2023-04-15T15:10:00.000+0000",
      "affectedCustomerCount": 29,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "ffb912b3-1f68-4900-9fb5-00c84f9b397b",
      "minLongitude": -1.0558,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -1.0527980251073408,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-04-15T14:30:00.000+0000",
      "estimatedRestoration": "2023-04-16T02:30:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. We are working hard to get the power back on as quickly as we can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KU2009'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KU2009",
      "maxLongitude": -1.0501,
      "minLatitude": 51.2935,
      "name": "RG24 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "RG24 8EX",
        "RG24 8RL",
        "RG24 8RS",
        "RG24 8RW",
        "RG24 8XJ"
      ],
      "location": {
        "coordinates": [
          [
            [
              -1.0558,
              51.2971
            ],
            [
              -1.0501,
              51.2971
            ],
            [
              -1.0501,
              51.2935
            ],
            [
              -1.0558,
              51.2935
            ],
            [
              -1.0558,
              51.2971
            ]
          ]
        ],
        "type": "Polygon"
      },
      "estimatedArrivalOnSiteTime": "2023-04-15T15:15:00.000+0000",
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 51.2971,
      "etrUpdate": true
    },
    {
      "latitude": 51.18320157520243,
      "type": "LV",
      "reference": "KU2173",
      "etaUpdate": true,
      "affectedCustomerCount": 7,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "64a524fa-c54c-48d4-b432-d877a5f3aa49",
      "minLongitude": -0.6237,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -0.619335486510898,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-04-15T21:19:00.000+0000",
      "estimatedRestoration": "2023-04-16T05:30:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. We are working hard to get the power back on as quickly as we can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KU2173'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KU2173",
      "maxLongitude": -0.617,
      "minLatitude": 51.1808,
      "name": "GU7 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "GU7 1QU",
        "GU7 1QX",
        "GU7 1QY",
        "GU7 1RQ"
      ],
      "location": {
        "coordinates": [
          [
            [
              -0.6237,
              51.1848
            ],
            [
              -0.617,
              51.1848
            ],
            [
              -0.617,
              51.1808
            ],
            [
              -0.6237,
              51.1808
            ],
            [
              -0.6237,
              51.1848
            ]
          ]
        ],
        "type": "Polygon"
      },
      "estimatedArrivalOnSiteTime": "2023-04-15T23:30:00.000+0000",
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 51.1848,
      "etrUpdate": true
    },
    {
      "latitude": 50.89833208641456,
      "type": "PSI",
      "reference": "215615",
      "etaUpdate": true,
      "affectedCustomerCount": 100,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "5368f318-7939-4202-ae35-c56149a25254",
      "minLongitude": -2.3571,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -2.3322498592628746,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-04-16T07:30:00.000+0000",
      "estimatedRestoration": "2023-04-22T15:30:00.000+0000",
      "message": "We're sorry for the loss of supply. The reason you're currently without power is a planned outage in the areas listed. We have turned off your electricity supply to allow us to safely carry out essential work to our electrical equipment. By doing this, it'll help us to improve the reliability of our service to you for the future. If you have any questions or require more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) with reference  '215615'. You will have received a letter from us confirming your power will be interrupted with specific information relating to your property. If you did not receive this letter, you can assume your power will not be interrupted.",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/215615",
      "maxLongitude": -2.3084,
      "minLatitude": 50.8804,
      "name": "DT10 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "DT10 2AF",
        "DT10 2AG",
        "DT10 2AJ",
        "DT10 2AL",
        "DT10 2AN",
        "DT10 2AP",
        "DT10 2AQ",
        "DT10 2AW",
        "DT10 2DW"
      ],
      "location": {
        "coordinates": [
          [
            [
              -2.3571,
              50.9148
            ],
            [
              -2.3084,
              50.9148
            ],
            [
              -2.3084,
              50.8804
            ],
            [
              -2.3571,
              50.8804
            ],
            [
              -2.3571,
              50.9148
            ]
          ]
        ],
        "type": "Polygon"
      },
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 50.9148,
      "etrUpdate": true
    },
    {
      "latitude": 50.89833208641456,
      "type": "PSI",
      "reference": "215616",
      "etaUpdate": true,
      "affectedCustomerCount": 100,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "932b4a7c-907c-423d-9dab-fc729f470efc",
      "minLongitude": -2.3571,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -2.3322498592628746,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-04-16T07:30:00.000+0000",
      "estimatedRestoration": "2023-04-22T15:30:00.000+0000",
      "message": "We're sorry for the loss of supply. The reason you're currently without power is a planned outage in the areas listed. We have turned off your electricity supply to allow us to safely carry out essential work to our electrical equipment. By doing this, it'll help us to improve the reliability of our service to you for the future. If you have any questions or require more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) with reference  '215616'. You will have received a letter from us confirming your power will be interrupted with specific information relating to your property. If you did not receive this letter, you can assume your power will not be interrupted.",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/215616",
      "maxLongitude": -2.3084,
      "minLatitude": 50.8804,
      "name": "DT10 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "DT10 2AF",
        "DT10 2AG",
        "DT10 2AJ",
        "DT10 2AL",
        "DT10 2AN",
        "DT10 2AP",
        "DT10 2AQ",
        "DT10 2AW",
        "DT10 2DW"
      ],
      "location": {
        "coordinates": [
          [
            [
              -2.3571,
              50.9148
            ],
            [
              -2.3084,
              50.9148
            ],
            [
              -2.3084,
              50.8804
            ],
            [
              -2.3571,
              50.8804
            ],
            [
              -2.3571,
              50.9148
            ]
          ]
        ],
        "type": "Polygon"
      },
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 50.9148,
      "etrUpdate": true
    },
    {
      "latitude": 50.838031434579456,
      "type": "HV",
      "reference": "KU2317",
      "etaUpdate": true,
      "affectedCustomerCount": 141,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "1580ff0b-30f2-4544-8f46-dac2021b3027",
      "minLongitude": -2.202,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -2.1469613788438044,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-04-16T08:04:00.000+0000",
      "estimatedRestoration": "2023-04-16T17:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KU2317'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KU2317",
      "maxLongitude": -2.1335,
      "minLatitude": 50.8264,
      "name": "DT11 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "DT11 0QB",
        "DT11 9FH",
        "DT11 9FJ",
        "DT11 9FT",
        "DT11 9NH",
        "DT11 9NQ",
        "DT11 9NX",
        "DT11 9NY",
        "DT11 9NZ",
        "DT11 9PA",
        "DT11 9PB",
        "DT11 9PD"
      ],
      "location": {
        "coordinates": [
          [
            [
              -2.202,
              50.879
            ],
            [
              -2.1335,
              50.879
            ],
            [
              -2.1335,
              50.8264
            ],
            [
              -2.202,
              50.8264
            ],
            [
              -2.202,
              50.879
            ]
          ]
        ],
        "type": "Polygon"
      },
      "estimatedArrivalOnSiteTime": "2023-04-16T08:45:00.000+0000",
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 50.879,
      "etrUpdate": true
    },
    {
      "latitude": 50.72185739602958,
      "type": "LV",
      "reference": "KU2359",
      "etaUpdate": true,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "cb2b394b-e4ee-455d-8c02-562854e753b6",
      "minLongitude": -2.0242,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -2.0193983258958537,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-04-16T08:45:00.000+0000",
      "estimatedRestoration": "2023-04-16T13:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KU2359'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KU2359",
      "maxLongitude": -2.0153,
      "minLatitude": 50.7187,
      "name": "BH15 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "BH15 4JY"
      ],
      "location": {
        "coordinates": [
          [
            [
              -2.0242,
              50.7243
            ],
            [
              -2.0153,
              50.7243
            ],
            [
              -2.0153,
              50.7187
            ],
            [
              -2.0242,
              50.7187
            ],
            [
              -2.0242,
              50.7243
            ]
          ]
        ],
        "type": "Polygon"
      },
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 50.7243,
      "etrUpdate": true
    },
    {
      "latitude": 50.725169658998574,
      "type": "LV",
      "reference": "KU2412",
      "etaUpdate": true,
      "engineerOnSiteTime": "2023-04-16T11:45:00.000+0000",
      "affectedCustomerCount": 22,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "11e0a415-1cf8-4304-935e-6d507488765c",
      "minLongitude": -1.2263,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -1.2229693686559633,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-04-16T10:58:00.000+0000",
      "estimatedRestoration": "2023-04-16T19:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KU2412'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KU2412",
      "maxLongitude": -1.2143,
      "minLatitude": 50.7139,
      "name": "PO33 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "PO33 4LA",
        "PO33 4LB",
        "PO33 4LE",
        "PO33 4LQ",
        "PO33 4PB"
      ],
      "location": {
        "coordinates": [
          [
            [
              -1.2263,
              50.7276
            ],
            [
              -1.2143,
              50.7276
            ],
            [
              -1.2143,
              50.7139
            ],
            [
              -1.2263,
              50.7139
            ],
            [
              -1.2263,
              50.7276
            ]
          ]
        ],
        "type": "Polygon"
      },
      "estimatedArrivalOnSiteTime": "2023-04-16T12:15:00.000+0000",
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 50.7276,
      "etrUpdate": true
    },
    {
      "latitude": 50.929968472397476,
      "type": "LV",
      "reference": "KU2438",
      "etaUpdate": true,
      "affectedCustomerCount": 22,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "af702045-bd0f-4514-bcc6-081a1954a9f2",
      "minLongitude": -1.3902,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -1.3877141375989124,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-04-16T11:18:00.000+0000",
      "estimatedRestoration": "2023-04-16T21:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KU2438'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KU2438",
      "maxLongitude": -1.3841,
      "minLatitude": 50.9272,
      "name": "SO17 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "SO17 1WZ",
        "SO17 2AG",
        "SO17 2AQ",
        "SO17 3AE",
        "SO17 3AG",
        "SO17 3SB"
      ],
      "location": {
        "coordinates": [
          [
            [
              -1.3902,
              50.9323
            ],
            [
              -1.3841,
              50.9323
            ],
            [
              -1.3841,
              50.9272
            ],
            [
              -1.3902,
              50.9272
            ],
            [
              -1.3902,
              50.9323
            ]
          ]
        ],
        "type": "Polygon"
      },
      "estimatedArrivalOnSiteTime": "2023-04-16T12:10:00.000+0000",
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 50.9323,
      "etrUpdate": true
    },
    {
      "latitude": 51.0806347266079,
      "type": "PSI",
      "reference": "215079",
      "etaUpdate": true,
      "affectedCustomerCount": 83,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "c9ccd23a-bde3-4d64-be3b-a59be9cba15b",
      "minLongitude": -0.7343,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -0.7289775701443679,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-04-16T12:00:00.000+0000",
      "estimatedRestoration": "2023-04-16T16:00:00.000+0000",
      "message": "We're sorry for the loss of supply. The reason you're currently without power is a planned outage in the areas listed. We have turned off your electricity supply to allow us to safely carry out essential work to our electrical equipment. By doing this, it'll help us to improve the reliability of our service to you for the future. If you have any questions or require more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) with reference  '215079'. You will have received a letter from us confirming your power will be interrupted with specific information relating to your property. If you did not receive this letter, you can assume your power will not be interrupted.",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/215079",
      "maxLongitude": -0.7151,
      "minLatitude": 51.0518,
      "name": "GU27 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "GU27 2PH",
        "GU27 2QP",
        "GU27 3AA",
        "GU27 3DA",
        "GU27 3DF",
        "GU27 3DG",
        "GU27 3DQ",
        "GU27 3LQ",
        "GU27 3SA",
        "GU27 3SD",
        "GU27 3SE",
        "GU27 3SF",
        "GU27 3SG",
        "GU27 3SJ",
        "GU27 3SQ",
        "GU27 3SU"
      ],
      "location": {
        "coordinates": [
          [
            [
              -0.7343,
              51.0852
            ],
            [
              -0.7151,
              51.0852
            ],
            [
              -0.7151,
              51.0518
            ],
            [
              -0.7343,
              51.0518
            ],
            [
              -0.7343,
              51.0852
            ]
          ]
        ],
        "type": "Polygon"
      },
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 51.0852,
      "etrUpdate": true
    },
    {
      "latitude": 51.28830900851655,
      "type": "LV",
      "reference": "KU1759",
      "etaUpdate": true,
      "affectedCustomerCount": 5,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "6baba1fb-fb64-4e64-9c20-3d6b43a99a95",
      "minLongitude": -0.9591,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -0.957805982149233,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-04-16T13:30:00.000+0000",
      "estimatedRestoration": "2023-04-16T18:30:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KU1759'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KU1759",
      "maxLongitude": -0.9561,
      "minLatitude": 51.2876,
      "name": "RG27 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "RG27 9RF",
        "RG27 9RG"
      ],
      "location": {
        "coordinates": [
          [
            [
              -0.9591,
              51.2893
            ],
            [
              -0.9561,
              51.2893
            ],
            [
              -0.9561,
              51.2876
            ],
            [
              -0.9591,
              51.2876
            ],
            [
              -0.9591,
              51.2893
            ]
          ]
        ],
        "type": "Polygon"
      },
      "estimatedArrivalOnSiteTime": "2023-04-16T08:30:00.000+0000",
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 51.2893,
      "etrUpdate": true
    },
    {
      "latitude": 50.83727903197265,
      "type": "LV",
      "reference": "KU2480",
      "etaUpdate": true,
      "affectedCustomerCount": 9,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "4e0a92b7-66b4-4eb8-812c-592efcfd55d3",
      "minLongitude": -2.1484,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -2.144587416723487,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-04-16T08:10:00.000+0000",
      "estimatedRestoration": "2023-04-16T17:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KU2480'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KU2480",
      "maxLongitude": -2.1415,
      "minLatitude": 50.8345,
      "name": "DT11 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "DT11 9LG",
        "DT11 9NF",
        "DT11 9NG",
        "DT11 9NY",
        "DT11 9PJ"
      ],
      "location": {
        "coordinates": [
          [
            [
              -2.1484,
              50.84
            ],
            [
              -2.1415,
              50.84
            ],
            [
              -2.1415,
              50.8345
            ],
            [
              -2.1484,
              50.8345
            ],
            [
              -2.1484,
              50.84
            ]
          ]
        ],
        "type": "Polygon"
      },
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 50.84,
      "etrUpdate": true
    },
    {
      "latitude": 51.35155481767378,
      "type": "LV",
      "reference": "KU2515",
      "etaUpdate": true,
      "affectedCustomerCount": 10,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "b171df96-2f3b-4a45-9445-e998fd703f74",
      "minLongitude": -1.1039,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -1.102858430317155,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-04-16T14:20:00.000+0000",
      "estimatedRestoration": "2023-04-16T20:30:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KU2515'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KU2515",
      "maxLongitude": -1.1015,
      "minLatitude": 51.3506,
      "name": "RG7 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "RG7 2QE"
      ],
      "location": {
        "coordinates": [
          [
            [
              -1.1039,
              51.3525
            ],
            [
              -1.1015,
              51.3525
            ],
            [
              -1.1015,
              51.3506
            ],
            [
              -1.1039,
              51.3506
            ],
            [
              -1.1039,
              51.3525
            ]
          ]
        ],
        "type": "Polygon"
      },
      "estimatedArrivalOnSiteTime": "2023-04-16T15:10:00.000+0000",
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 51.3525,
      "etrUpdate": true
    },
    {
      "latitude": 50.74934341282088,
      "type": "LV",
      "reference": "KU2147",
      "etaUpdate": true,
      "engineerOnSiteTime": "2023-04-15T21:05:00.000+0000",
      "affectedCustomerCount": 7,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "201a769b-7727-48ab-a857-79e8724f5ca3",
      "minLongitude": -1.7167,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -1.715082310327418,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-04-15T13:55:00.000+0000",
      "estimatedRestoration": "2023-04-16T18:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KU2147'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KU2147",
      "maxLongitude": -1.7123,
      "minLatitude": 50.7478,
      "name": "BH23 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "BH23 4RL",
        "BH23 4RW"
      ],
      "location": {
        "coordinates": [
          [
            [
              -1.7167,
              50.7506
            ],
            [
              -1.7123,
              50.7506
            ],
            [
              -1.7123,
              50.7478
            ],
            [
              -1.7167,
              50.7478
            ],
            [
              -1.7167,
              50.7506
            ]
          ]
        ],
        "type": "Polygon"
      },
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 50.7506,
      "etrUpdate": true
    },
    {
      "latitude": 51.18639301398724,
      "type": "LV",
      "reference": "KU2528",
      "etaUpdate": true,
      "affectedCustomerCount": 18,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "460007c8-6d58-45e9-8b77-a672a9589c57",
      "minLongitude": -1.5126,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -1.4871437761162087,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-04-16T14:42:00.000+0000",
      "estimatedRestoration": "2023-04-16T22:59:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KU2528'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KU2528",
      "maxLongitude": -1.4781,
      "minLatitude": 51.1686,
      "name": "SP11 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "SP11 0JF",
        "SP11 7RH",
        "SP11 7RJ",
        "SP11 7RL",
        "SP11 7RN",
        "SP11 7RS",
        "SP11 7RX",
        "SP11 7RZ"
      ],
      "location": {
        "coordinates": [
          [
            [
              -1.5126,
              51.2762
            ],
            [
              -1.4781,
              51.2762
            ],
            [
              -1.4781,
              51.1686
            ],
            [
              -1.5126,
              51.1686
            ],
            [
              -1.5126,
              51.2762
            ]
          ]
        ],
        "type": "Polygon"
      },
      "estimatedArrivalOnSiteTime": "2023-04-16T16:00:00.000+0000",
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 51.2762,
      "etrUpdate": true
    },
    {
      "latitude": 50.81787884296142,
      "type": "LV",
      "reference": "KU2532",
      "etaUpdate": true,
      "affectedCustomerCount": 8,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "8c54ca53-fef4-4d4e-8716-b64ad3d762cf",
      "minLongitude": -0.8125,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -0.8024462721444846,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-04-16T11:00:00.000+0000",
      "estimatedRestoration": "2023-04-16T20:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KU2532'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KU2532",
      "maxLongitude": -0.794,
      "minLatitude": 50.8098,
      "name": "PO20 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "PO20 7DY",
        "PO20 7DZ",
        "PO20 7EA",
        "PO20 7EB",
        "PO20 7EQ"
      ],
      "location": {
        "coordinates": [
          [
            [
              -0.8125,
              50.823
            ],
            [
              -0.794,
              50.823
            ],
            [
              -0.794,
              50.8098
            ],
            [
              -0.8125,
              50.8098
            ],
            [
              -0.8125,
              50.823
            ]
          ]
        ],
        "type": "Polygon"
      },
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 50.823,
      "etrUpdate": true
    },
    {
      "latitude": 57.13858344414344,
      "type": "LV",
      "reference": "KU2538",
      "etaUpdate": true,
      "affectedCustomerCount": 25,
      "networkId": "com.sse.ssepd.shepd",
      "UUID": "749acc98-8780-4803-aff6-66060a1281d7",
      "minLongitude": -2.1236,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -2.121770146800918,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-04-16T15:07:00.000+0000",
      "estimatedRestoration": "2023-04-16T18:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KU2538'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KU2538",
      "maxLongitude": -2.1196,
      "minLatitude": 57.1374,
      "name": "AB10 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "AB10 6QY",
        "AB10 6QZ",
        "AB10 6RB",
        "AB10 6RE",
        "AB10 6RH"
      ],
      "location": {
        "coordinates": [
          [
            [
              -2.1236,
              57.1396
            ],
            [
              -2.1196,
              57.1396
            ],
            [
              -2.1196,
              57.1374
            ],
            [
              -2.1236,
              57.1374
            ],
            [
              -2.1236,
              57.1396
            ]
          ]
        ],
        "type": "Polygon"
      },
      "estimatedArrivalOnSiteTime": "2023-04-16T16:00:00.000+0000",
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 57.1396,
      "etrUpdate": true
    }
  ]
}
